package.json

```diff
@@ -139,20 +139,20 @@
     "@cliqz/adblocker-webextension-cosmetics": "^1.20.0",
     "@cliqz/url-parser": "^1.1.4",
     "abortcontroller-polyfill": "^1.5.0",
-    "anonymous-credentials": "https://github.com/cliqz-oss/anonymous-credentials/releases/download/1.0.0/anonymous-credentials-1.0.0.tgz",
+    "anonymous-credentials": "https://github.com/whotracksme/anonymous-credentials/releases/download/1.0.0/anonymous-credentials-1.0.0.tgz",
     "fast-deep-equal": "^3.1.3",
     "jsonschema": "^1.4.0",
-    "math-expression-evaluator": "^1.3.4",
+    "math-expression-evaluator": "^1.3.8",
     "moment": "^2.29.1",
     "node-fetch": "^2.6.1",
     "node-forge": "^0.10.0",
-    "pako": "^2.0.2",
+    "pako": "^2.0.4",
     "punycode": "^2.1.1",
-    "rusha": "^0.8.13",
-    "rxjs": "^6.6.3",
+    "rusha": "^0.8.14",
+    "rxjs": "^7.5.2",
     "text-encoding": "^0.7.0",
     "tldts-experimental": "^5.6.74",
-    "ua-parser-js": "^0.7.22",
+    "ua-parser-js": "^1.0.2",
```

Comment:
are we sure there are no breaking changes?

Reply:
For that, I don't think it will break something. Outside of anolysis, I'm not even sure if the old code uses it.

The main ghostery-extension also already uses 1.0.2.

Reply:
Regarding anolysis, I think the test suite is extensive enough to give good confidence about the change.

Reply:
For Anolysis, I had to change the test as that user agent is now detected as xbox:

```
'Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; Trident/6.0; Xbox; Xbox One'
```

But from my understanding, not detecting it as Desktop Window is an improvement. Apart from that, only the new PR for human-web uses ua-parser. Overall, upgrading that should be harmless. Perhaps I should create a small PR that we have the easy ones in before starting with potentially breaking ones.